java/internalmetrics.md

```diff
@@ -0,0 +1,185 @@
+|~sdk.java.core.sender.proxy.histograms.valid.count                         |Counter    |Valid distributions received.|
+|~sdk.java.core.sender.proxy.histograms.invalid.count                       |Counter    |Invalid distributions received.|
+|~sdk.java.core.sender.proxy.histograms.dropped.count                       |Counter    |Distributions dropped due to failure to write to socket.|
+|~sdk.java.core.sender.proxy.histogramHandler.errors.count                  |Counter    |Errors encountered, excluding connection errors. |
+|~sdk.java.core.sender.proxy.histogramHandler.connect.errors.count          |Counter    |Errors encountered connecting to remote socket.|
+|~sdk.java.core.sender.proxy.histogramHandler.socket.write.success.count    |Counter    |Successful writes to socket.|
+|~sdk.java.core.sender.proxy.histogramHandler.socket.write.errors.count     |Counter    |Unsuccessful writes to socket.|
+|~sdk.java.core.sender.proxy.histogramHandler.socket.flush.success.count    |Counter    |Successful socket flushes.|
+|~sdk.java.core.sender.proxy.histogramHandler.socket.flush.errors.count     |Counter    |Unsuccessful socket flushes.|
+|~sdk.java.core.sender.proxy.histogramHandler.socket.reset.success.count    |Counter    |Successful connection resets.|
+|~sdk.java.core.sender.proxy.histogramHandler.socket.reset.errors.count     |Counter    |Unsuccessful connection resets.|
+
+### Tracing Handler
+|Metric Name|Metric Type|Description|
+|:---|:---:|:---|
+|~sdk.java.core.sender.proxy.spans.discarded.count                      |Counter    |Spans discarded due to port(s) that were not configured.|
+|~sdk.java.core.sender.proxy.spans.valid.count                          |Counter    |Valid spans received.|
+|~sdk.java.core.sender.proxy.spans.invalid.count                        |Counter    |Invalid spans received.|
+|~sdk.java.core.sender.proxy.spans.dropped.count                        |Counter    |Spans dropped due to failure to write to socket.|
+|~sdk.java.core.sender.proxy.tracingHandler.errors.count                |Counter    |Errors encountered, excluding connection errors. |
+|~sdk.java.core.sender.proxy.tracingHandler.connect.errors.count        |Counter    |Errors encountered connecting to remote socket.|
+|~sdk.java.core.sender.proxy.tracingHandler.socket.write.success.count  |Counter    |Successful writes to socket.|
+|~sdk.java.core.sender.proxy.tracingHandler.socket.write.errors.count   |Counter    |Unsuccessful writes to socket.|
+|~sdk.java.core.sender.proxy.tracingHandler.socket.flush.success.count  |Counter    |Successful socket flushes.|
+|~sdk.java.core.sender.proxy.tracingHandler.socket.flush.errors.count   |Counter    |Unsuccessful socket flushes.|
+|~sdk.java.core.sender.proxy.tracingHandler.socket.reset.success.count  |Counter    |Successful connection resets.|
+|~sdk.java.core.sender.proxy.tracingHandler.socket.reset.errors.count   |Counter    |Unsuccessful connection resets.|
+
+## Metrics Collected When Using Direct Ingestion
```

Comment:
I would start this with a sort-of disclaimer, like "During Proof-of_Concept implementations, some developers send message directly to the Wavefront service." 
And, start the proxy part with the opposite disclaimer.

Reply:
Will move back to address this later on, as discussed.